eras/conway/impl/src/Cardano/Ledger/Conway/PParams.hs

```diff
@@ -260,7 +258,7 @@ data UpgradeConwayPParams f = UpgradeConwayPParams
   { ucppPoolVotingThresholds :: !(HKD f PoolVotingThresholds)
   , ucppDRepVotingThresholds :: !(HKD f DRepVotingThresholds)
   , ucppCommitteeMinSize :: !(HKD f Natural)
-  , ucppCommitteeMaxTermLength :: !(HKD f Natural)
+  , ucppCommitteeMaxTermLength :: !(HKD f EpochNo)
```

Comment:
like I said above, it should not be a point in time, but an interval

Reply:
Agreed. I've captured the proper fix in this ticket: https://github.com/input-output-hk/cardano-ledger/issues/3870